### DISCUSSION TITLE
2D vs 3D ?

### DISCUSSION THREAD

**Discussion by ryankmims** [Godot-Community-Games-Org/game1#3] [Polls]
As people join I think it'd be good to figure out where everyone stands in terms of what they're comfortable/familiar with.

As ideas roll in, it would be good to make limitations if we need to (ie. if an idea comes in that really only works in 3D, but 95% of the contributors are familiar with 2D, then we should probably find a different idea, or rethink the idea in two dimensions)

**Comment by GodotCommunityGames** [comment]
Yes absolutely. As we get to the point of discussing game ideas we may find the community decides on a few different ideas to get started with, with some people only interested in certain ideas. I think likely it'd make sense to at very least get one 2D game and one 3D game going at the start since no doubt there will be some people only with experience/interest in one or the other.
